### ISSUE TITLE
Amount exceeds maximum amount allowed Error 

### ISSUE THREAD

**Issue by ashok1421** [razorpay/razorpay-magento#490]
Hello @yashgit891 ,
I am using the Razor pay module version 4.1.4
On Checkout page Payment Popup not appear  in our website and i checked in network tab it showed error 
**{"message":"Amount exceeds maximum amount allowed.","parameters":[]}**

I found the error in my razorpay.log file.
```
Razorpay Order: create order started with Magento OrderId:242500011 and amount:59683600 [] []
[2024-05-19 09:31:49] main.CRITICAL: Razorpay Order: Mage Order(242500011), Error message from API: Amount exceeds the maximum amount allowed. [] []
[2024-05-19 09:31:49] main.INFO: Data saved in razorpay_sales_order for Mage Order(242500011) [] []
```
Order placed with pending status. 
Please check the Attached File.
![Screenshot 2024-05-21 at 12 35 01 PM](https://github.com/razorpay/razorpay-magento/assets/75234796/b2cdf7be-6624-440d-b974-036d16f09e1a)

**Comment by bhaveshseepossible** [comment]
The same issue is replicated in local with test credentials.

<img width="2560" alt="Screenshot 2024-05-21 at 1 04 02 PM" src="https://github.com/razorpay/razorpay-magento/assets/115447202/901d2c1e-b46a-42db-b178-33f549abdba1">

Thank you.

**Comment by yashgit891** [comment]
@ashok1421 In order grid the order is present as Magento allows order creation for any amount. Post magento order creation, when Razorpay order is being created it is rejecting the order as amount is too large. To increase this amount threshold please reach out to the support team as that is merchant level configuration.

**Comment by ashok1421** [comment]
@yashgit891 Please Manage This error as Display Error Message .

**Comment by bhaveshseepossible** [comment]
@yashgit891 yes you are absolutely right but in the production environment when the customer see nothing after clicking on the place order button the customer cannot understand what is happening and this is very bad customer experience so you should manage this error on frontend and at list show message that this much amount is not applicable with Razorpay so try another payment method before placing this order. now the code is placing an order and the customer is not aware that the order is placed without payment on this page. so it should be managed here.

Thank you.

**Comment by yashgit891** [comment]
@ashok1421 @bhaveshseepossible marking this as an enhancement.
